web/modules/custom/dpl_update/dpl_update.install

```diff
@@ -910,3 +925,15 @@ function dpl_update_update_10060(): string {
     'http_cache_control.settings',
   ]);
 }
+
+/**
+ * Add the "Hjælp til eReolen" subscription to BNF.
+ *
+ * @throws \Exception
```

Comment:
Please remove this. 

Generally we do not declare exceptions except when we have a specific reason to mention them. This is not the case here.

I have tried to check if there is a way to detect/prevent this but I have not found anything yet.

Reply:
I removed it in the rebase